hack/pin-custom-bundle-dockerfile.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+cp custom-bundle.Dockerfile custom-bundle.Dockerfile.pinned
+#loop over each openstack-k8s-operators go.mod entry
+for X in $(go list -m -json all | jq -r '. | select(.Path | contains("openstack")) | "\(.Path)=\(.Version)"'); do
```

Comment:
This looks like it should be fine, but running it locally on this repo I seem to be getting an error:
```
[fedora@fedora-work openstack-operator]$ go list -m -json all
go: github.com/openshift/api@v3.9.0+incompatible: invalid version: unknown revision v3.9.0
go: github.com/openshift/api@v3.9.0+incompatible: reading github.com/openshift/api/go.mod at revision v3.9.0: unknown revision v3.9.0
```

Is this a scenario we need to handle in this script?

Reply:
what 'go version' are you using locally? We require v1.18.x. I'm at go version go1.18.6 linux/amd64

Reply:
The one that doesn't work is Fedora with a slightly different minor version to you:
```
go version go1.18.7 linux/amd64
```

But interestingly, it works when I try it on my Mac which is:
```
❯ go version
go version go1.19.2 darwin/arm64
```

Not sure what the deal is there. But if we're happy that we should never need to handle that, we should be good here.